monix-java/src/main/scala/monix/java8/execution/package.scala

```diff
@@ -31,7 +31,6 @@ package object execution {
     * DEPRECATED — switch to Scala 2.12+ and
     * [[monix.execution.CancelableFuture.fromJavaCompletable CancelableFuture.fromJavaCompletable]].
     */
-  @deprecated("Switch to Scala 2.12+ and CancelableFuture.fromJavaCompletable", "3.0.0")
```

Comment:
Why is this undeprecated?

Reply:
Same as the others — Scala 3 generates warnings, even if the extensions don't get used, which is why I prefer to have the deprecation warnings only on the methods themselves.